Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -119,7 +119,7 @@ private void GenerateCodeForParts(StringBuilder codeBuilder, IEnumerable<Content
             }
         }
 
-        private string ConvertJToken(JToken jToken)
+        private string ConvertJToken(JToken jToken, int indentationDepth)

```

Comment:
Please open this project from [OSOCE](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/) so you can see the analyzer violations:

![image](https://user-images.githubusercontent.com/1976647/157293990-8bd40c43-8168-4894-8faa-e9e0c7bd3b94.png)

Reply:
This is great! I didn't know this existed. I'll get it up and running and take a look.

Reply:
I cloned the solution, replaced the existing one with my version of Helpful-Extensions but I'm not getting any warnings. Am I missing anything?

Reply:
Hmm, you really should see them, and without doing anything special. Did you clone with submodules? Can you build the solution?

Reply:
Nevermind I got it now. Updating.